README.md

```diff
@@ -100,14 +100,21 @@ Open the rule in SonarQube / SonarCloud, scroll down and (in case the rule has p
 
 Use SonarLint Connected Mode to connect to SonarQube and SonarCloud.
 
-### standalone nuget / SonarLint
+### SonarLint
 
-You can see how we do it in our own repository for SonarLint Connected mode (and you can reuse the same concept for the nuget package):
+The easiest way is to configure a Quality Profile in SonarCloud.
+
+* Create a dummy repository and analyze it in SonarCloud (it's free for open-source).
+* Configure the Quality Profile for the project you created.
+* Then connect SonarLint to that project, and it will configure itself (with a ruleset and SonarLint.xml file) locally based on the Quality Profile.

```

Comment:
Just to be more explicit for the NuGet stage
```suggestion
* Then connect SonarLint to that project, and it will download the configuration (ruleset and SonarLint.xml files) locally and update your project based on the Quality Profile.
```